packages/clay-management-toolbar/src/ClayManagementToolbar.soy

```diff
@@ -757,7 +762,7 @@
 				action="{$searchActionURL}"
 			{/if}
 
-			method="POST"
+			method="{$searchFormMethod}"
```

Comment:
Because a default value is defined in the `js` component we need to to the same here, so this should be:
`{$searchFormMethod ?: 'GET'}`